### PR TITLE
Fix link to API Url and customize components

### PR DESCRIPTION
**Pull request by nishimweprince**
## What does this PR do?

This PR refactors our code to hide the backend API url in environment variables and extends components customizability

## Description of Task to be completed?

+ Update the link to the backend after redeploying the database
+ Move sensitive information to environment variables
+ Update components

## How should this be manually tested?

- Clone the repository using `git clone https://github.com/atlp-rwanda/e-comm-team-techsmith-fn`

- Checkout to the branch using `git checkout bug-update-variables`

- Install node packages that the application depends on: `npm i`

- Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.

- Start the application development server: `npm start`

- Check out the components documentation: `npm run storybook`

## Any background context you want to provide?

All environment variables must start with `VITE` to exposed to the Vite processed code. Refer to [Vite's documentation](https://vitejs.dev/guide/env-and-mode.html) to learn more.

### CODE REVIEW

**Review by frankflej — APPROVED** [review]
Perfect!

**Review by jcharles921 — APPROVED** [review]

File: src/containers/LoginContainer.jsx [hunk 1]
```diff
@@ -13,6 +13,7 @@ import { useNavigate } from 'react-router-dom';
 import Loading from '../components/Loading';
 import { useDispatch } from 'react-redux';
 import { getToken } from '../states/features/login/actions';
+import { API_URL } from '../constants';
```
Comment (niyontwali) [inline src/containers/LoginContainer.jsx:16]: Why is it comming from the constants though, why not have it in the `.env` file?

**Review by niyontwali — CHANGES_REQUESTED** [review]

**Review by irakevine — APPROVED** [review]
Great Work

File: src/containers/LoginContainer.jsx (hunk 1 above)
Comment (nishimweprince) [reply src/containers/LoginContainer.jsx:16]: We imported it from the constants to allow ourselves to be change it from one place and update in the rest of the code. [This is where we loaded it from the .env](https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/blob/bug-update-variables/src/constants/API.jsx#L1). 

```js
const API_URL = import.meta.env.VITE_API_URL;

export { API_URL }
```

**Review by cyubahiro367 — APPROVED** [review]

**Review by niyontwali — APPROVED** [review]

**Review by bergad37 — APPROVED** [review]
Well done!

**Pull request merged**